src/engine/qcommon/qcommon.h

```diff
@@ -285,7 +288,7 @@ You or the server may be running older versions of the game."
 
 #define PROTOCOL_VERSION       86
 
-#define URI_SCHEME             GAMENAME_STRING "://"
+#define URI_SCHEME             "dgp://"
```

Comment:
External tools would only need to call the engine, the game for the server should be part of the out of band protocol, I think getstatus shows it.